tests/integration/cartography/intel/aws/test_lambda.py

```diff
@@ -1,5 +1,7 @@
 import cartography.intel.aws.lambda_function
 import tests.data.aws.lambda_function
+from tests.integration.util import check_nodes
+from tests.integration.util import check_rels
```

Comment:
Can you add a test that covers the whole sync function where the `get` is mocked out like with other refactors?